<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maincircuito</class>
 <widget class="QMainWindow" name="maincircuito">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador de Circuitos Digitais</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="portasConexoesWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>350</width>
      <height>481</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>TIPO</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>N° ENTR</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ENTR 1</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ENTR 2</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ENTR 3</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ENTR 4</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="saidasWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>49</y>
      <width>120</width>
      <height>481</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <column>
     <property name="text">
      <string>ORIG/SAIDA</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="portasConexoesLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PORTAS E CONEXÕES</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="saidasLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SAÍDAS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelaVerdadeWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>450</width>
      <height>500</height>
     </rect>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tabelaVerdadeLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>450</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TABELA VERDADE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="circuitosLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>470</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CIRCUITOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCircuito">
    <property name="title">
     <string>Circuito</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionLer"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuSimular">
    <property name="title">
     <string>Simular</string>
    </property>
    <addaction name="actionGerar_tabela"/>
   </widget>
   <addaction name="menuCircuito"/>
   <addaction name="menuSimular"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo...</string>
   </property>
  </action>
  <action name="actionLer">
   <property name="text">
    <string>Ler...</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar...</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionGerar_tabela">
   <property name="text">
    <string>Gerar tabela</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
